tools/spades_header_fixer/spades_header_fixer.xml

```diff
@@ -3,7 +3,7 @@
   <requirements>
       <requirement type="package" version="4.7.0">sed</requirement>
   </requirements>
-	<command >sed -r "s/>NODE(_[0-9]+)_(.*)/>${input.element_identifier}\1 \2/g" $input >$output</command>
+  <command >sed -r "s/>NODE(_[0-9]+)_(.*)|>contig/>${input.element_identifier}\1 \2/g" $input >$output</command>
```

Comment:
Thanks for the fix! Are you able to add in a second test in the `tests` block to check that this behaviour is working as intended?